.gitignore

```diff
@@ -0,0 +1,2 @@
+input.conf
+output.conf
```

Comment:
I think lint will fail here **No newline at the end of file**

Reply:
Thanks! Fixed.